include/seahorn/OperationalSemantics.hh

```diff
@@ -179,6 +185,12 @@ struct FunctionInfo {
 };
 
 class OperationalSemantics {
+private:
+  //// \brief crab's cfg builder manager
+  std::unique_ptr<clam::CrabBuilderManager> m_cfg_builder_man;
+  //// \brief crab instance to solve alloc bounds
+  std::unique_ptr<clam::InterGlobalClam> m_crab_rng_solver;
```

Comment:
I don't see that we set it to `nullptr` in the constructor.

Reply:
Thanks for the correction.